particles/Services/schemas/RandomData.schema

```diff
@@ -0,0 +1,5 @@
+// Wrapper around a random number so that we can input this into
+// Recipes
+
+schema RandomData
+  Number next
```

Comment:
Rename this to something like `value` ?

Reply:
Alternative is to emulate WebCrypto `getRandomValues`

https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues